.gitignore

```diff
@@ -22,11 +22,9 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-/dist
```

Comment:
This PR would be ready if this change was removed, that way all the `dist` code that was checked would be deleted